server/server.js

```diff
@@ -4,8 +4,6 @@ const app = express();
 const port = 3001;
 const db = require('../db/index.js');
 
-app.use(express.static('client/dist'));
```

Comment:
you may want to use the built in dirname from express in the path of your static files location. the path may change once it deployed to amazon.

Reply:
great called, fixed that. Thanks!